.github/workflows/release.yaml

```
@@ -99,7 +99,9 @@ jobs:
       if: ${{ startsWith(github.ref, 'refs/tags/') }}
       with:
         draft: false
-        files: ${{ steps.package.outputs.archive }}
+        files: |
+          ${{ steps.package.outputs.archive }}
+          ${{ steps.package.outputs.archive-checksum }}
```

Can you avoid needing to create another output variable by doing this:

```suggestion
          ${{ steps.package.outputs.archive }}.sha256sum
```